search/src/main/res/layout/fragment_default_search_suggestions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".DefaultSearchSuggestionsFragment">
+
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="Default search suggestions" />
```

Comment:
This'll go into a R.string resource later, correct?

Reply:
I am solving this on different pull req. This was used as the dummy element.

Reply:
Cool, that's fine. I'll resolve this discussion.